tests/test_rank_models.py

```
@@ -0,0 +1,330 @@
+        assert u > 0  # doesn't crash, clamped
+
+
+# ---------------------------------------------------------------------------
+# TestComputeIntelligence
+# ---------------------------------------------------------------------------
+
+
+class TestComputeIntelligence:
+    def test_known_model_coding(self):
+        bench = _make_bench()
+        I = rank_models.compute_intelligence("gemini-3-flash", "coding", bench)
```

Ruff (0.15.18)

[error] Ambiguous variable name: `I`

(E741)